crates/src/grpc/grpc_subscriber.rs

```diff
@@ -381,6 +405,38 @@ impl DriftGrpcClient {
                                     }
                                 }
                             }
+                            Some(UpdateOneof::Transaction(tx_update)) => {
+                                let tx = match tx_update.transaction {
+                                    Some(ref tx) => tx,
+                                    None => {
+                                        warn!(target: "grpc", "empty transaction update: {tx_update:?}");
+                                        continue;
+                                    }
+                                };
+                                let transaction = match tx.transaction {
+                                    Some(ref tx) => tx,
+                                    None => {
+                                        warn!(target: "grpc", "empty transaction update: {tx_update:?}");
+                                        continue;
+                                    }
+                                };
+                                let meta = match tx.meta {
+                                    Some(ref meta) => meta,
+                                    None => {
+                                        warn!(target: "grpc", "empty transaction meta: {tx_update:?}");
+                                        continue;
+                                    }
+                                };
```

Comment:
wondering what happens if valid tx but has no meta? not sure if that'll happen in practice

Reply:
yeah i dont think it'll happen in practice, but i feel it's safer to check. I saw a surprising number of tx logs without signatures (11111...) from websocket, but none were like that from grpc